### PR TITLE
Stability updates + layout

### PR DESCRIPTION
**Pull request by nleroy917**
This includes a lot of stability improvements after the transition to React. Thanks for pointing out these things @Khoroshevskyi 

Improvements include:
1. Change key-binding on global search
2. Drag-n-drop was broken, it should now be fixed
3. Files were not getting uploaded properly for project submission
4. Sample table editing was broken due to trailing commas
5. New project page layout
6. Some new components + CSS utility classes to make front-end development easier
7. Sort project list by default to most recent first in the list

### CODE REVIEW

File: pephub/routers/api/v1/namespace.py [hunk 1]
```diff
@@ -63,13 +63,17 @@ async def get_namespace_projects(
         search_result = agent.annotation.get(
             namespace=namespace, limit=limit, offset=offset, admin=namespace_access
         )
+    results = [p.dict() for p in search_result.results]
+
+    # sort by last_update_date
```
Comment (khoroshevskyi) [inline pephub/routers/api/v1/namespace.py:68]: 1) if action in next line is explicit, we shouldn't add any comments. I think sorting function is understandable:)
2) This functionality should be in pepdbagent, not here. I wiil create Issue on this topic.

**Review by khoroshevskyi — CHANGES_REQUESTED** [review]
There is few things that still have to be polished: 
1) Download zip 3 different places. I think this link (or downloading button) should be in one place. E.g under "Get project" drop button. 
2) We don't have https downloading link (only zip) so I wouldn't add it to our UI now.
image

3) PEP version shows incorrect version:
image

4) Add subtables to UI. In my opinion everything is ready for adding subsamples. + I just added functionality for updating subsample tables. So it shouldn't be big problem.

Everything else looks good:)

**Comment by nleroy917** [comment]
Would be ideal to address these issues prior to merge + release:
- #143 
- #142 
- #141 
- #140 
- #131

**Comment by nleroy917** [comment]
@Khoroshevskyi Changes should be nearly complete now if you wanted to test it out.

**Pull request merged**